packages/docs/components/SideNav/SideNav.tsx

```diff
@@ -28,6 +29,8 @@ export const SideNav: React.FC = () => {
           <SideNavLink href="/GettingStarted/GettingStartedPage" as="/">
             Getting Started
           </SideNavLink>
+
+          <Link url={CodeSandboxUrl} title="CodeSandbox Example" />
```

Comment:
🍹What about adding this to the `Getting Started` page instead of adding a link here? Or we could have both the link here, but a description about the sample app on the `Getting Started` page?

🍹Also, we may want to use the `external` link variant. Would require some modification to the `Link` below.